clients/fides-js/src/lib/consent.tsx

```diff
@@ -1,10 +1,9 @@
 import { h, render } from "preact";
 
-import { ComponentType } from "./consent-types";
+import { ComponentType, FIDES_MODAL_LINK } from "./consent-types";
```

Comment:
Feels like this should mirror the `overlayParentElementId` option, no? Instead of a constant?

Reply:
yes good idea